GoogleDataTransport.podspec

```diff
@@ -0,0 +1,55 @@
+Pod::Spec.new do |s|
+  s.name             = 'GoogleDataTransport'
+  s.version          = '0.1.0'
+  s.summary          = 'Google iOS SDK data transport.'
+
+  s.description      = <<-DESC
+Shared library for iOS SDK data transport needs.
+                       DESC
+
+  s.homepage         = 'https://developers.google.com/'
+  s.license          = { :type => 'Apache', :file => 'LICENSE' }
+  s.authors          = 'Google, Inc.'
+  s.source           = {
+    :git => 'https://github.com/firebase/firebase-ios-sdk.git',
+    :tag => 'GoogleDataTransport-' + s.version.to_s
+  }
+
+  s.ios.deployment_target = '8.0'
+
+  # To develop or run the tests, >= 1.6.0 must be installed.
+  s.cocoapods_version = '>= 1.5.3'
```

Comment:
Can this be 1.4.0 to be consistent with the Firebase pods? It didn't seem compelling to update since 1.4.0 seem to still be working.